test/spmd/test_spmd_debugging.py

```diff
@@ -0,0 +1,205 @@
+import sys
+
+import unittest
+from unittest.mock import patch
+import math
+import numpy as np
+import os
+import io
+import rich
+
+import torch
+import torch_xla
+import torch_xla.runtime as xr
+import torch_xla.utils.utils as xu
+import torch_xla.core.xla_env_vars as xenv
+import torch_xla.core.xla_model as xm
+import torch_xla.experimental.xla_sharding as xs
```

Comment:
use `import torch_xla.distributed.spmd as xs` instead. Same thing with the other SPMD related imports.